src/user/user.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { UserModel } from 'src/models/user.model';
+
+@Injectable()
+export class UserService {
+  constructor(@InjectModel(UserModel) private userModel: typeof UserModel) {}
+
+  async createUser(id: number, username: string, password: string) {
+    const newUser = await this.userModel.create({ id, username, password });
+    return newUser;
+  }
+
+  async getAllUsers() {
+    try {
```

Comment:
Dont use try catch in the service, try catch should only appear in the controller and the controller will catch any errors propagated by the service